ownCloud/Viewer/DisplayViewController.swift

```diff
@@ -159,7 +164,8 @@ class DisplayViewController: UIViewController {
 
 			metadataInfoLabel!.centerXAnchor.constraint(equalTo: iconImageView.centerXAnchor),
 			metadataInfoLabel!.topAnchor.constraint(equalTo: iconImageView!.bottomAnchor, constant: 10),
-			metadataInfoLabel!.widthAnchor.constraint(equalTo: iconImageView.widthAnchor),
+			metadataInfoLabel!.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor, constant: 10),
```

Comment:
@pablocarmu may be use let constant instead of plain value 10?

Reply:
Sure, I'll do it!